cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -92,6 +92,10 @@ spec:
           - name: "WEBHOOK_SERVICE_PORT"
             value: "9443"
           {{- end }}
+          {{- if has "--enable-external-secret-stores" .Values.args }}
+          - name: "ESS_GENERATE_CERTS"
```

Comment:
I remember we discussed this but now I feel like it would be cleaner if we pass the name of the secret names and use them as a feature flag to decide whether the initializer should generate certs or not.

I mean:

```yaml
          - name: "ESS_CLIENT_CERTS_SECRET_NAME"
            value: ess-client-certs
          - name: "ESS_SERVER_CERTS_SECRET_NAME"
            value: ess-server-certs
```